### DISCUSSION TITLE
Webdriver could be used for authentication.

### DISCUSSION THREAD

**Discussion by SantiiRepair** [KRTirtho/spotube#967] [Ideas]
Using [webdriver.dart](https://github.com/google/webdriver.dart) to launch an instance of the browser and request it to authenticate with Spotify, then get the cookies instead of fetching the cookies manually, would be a friendlier way.

@KRTirtho what do you think?

**Comment by KRTirtho** [comment]
That's a brilliant idea. I'll test it out and check if it'll work.

**Reply by SantiiRepair** [reply]
I could do it and then open a pull request

**Reply by KRTirtho** [reply]
Yeah, sure. I'll be happy to review it.

**Reply by SantiiRepair** [reply]
[Here is an update](https://drive.google.com/file/d/16yVEvZeNvurvsXt9IyXYyRQ3NqUHk7sL/preview)
